### PR TITLE
Fix Bug in transfer all NEP5 asset, add some watch only filter

### PR DESCRIPTION
**Pull request by Lichen9618**
Bug fix for [Neo Issue 337](https://github.com/neo-project/neo/issues/337)

Adjust NetFee according to [Neo-Cli #378](https://github.com/neo-project/neo-cli/commit/e04e881814415d9fb4f887279aa48f4d33165eac)

Add some watch only filter

### CODE REVIEW

File: neo-gui/UI/NetFeeDialog.cs [hunk 1]
```diff
@@ -17,18 +17,15 @@ public partial class NetFeeDialog : Form
         Fixed8 NetFee;
         Fixed8 PriorityFee;
```
Comment (shargon) [inline neo-gui/UI/NetFeeDialog.cs]: Is this var used now?

Comment (Lichen9618) [reply neo-gui/UI/NetFeeDialog.cs]: @shargon 
`Fixed8 PriorityFee;` Not used now. I will remove it.

Thanks.

File: neo-gui/UI/InvokeContractDialog.cs [hunk 2]
```diff
@@ -33,53 +33,38 @@ public InvokeContractDialog(InvocationTransaction tx = null)
             }
         }
 
-        public InvocationTransaction GetTransaction()
+        public InvocationTransaction GetTransaction(Fixed8 fee, UInt160 Change_Address = null)
         {
-            Fixed8 fee = Fixed8.Zero;
-            if (tx.Size > 1024)
+            if (fee != Fixed8.Zero) { }
```
Comment (shargon) [inline neo-gui/UI/InvokeContractDialog.cs]: empty if?

Comment (Lichen9618) [reply neo-gui/UI/InvokeContractDialog.cs]: Right, unnecessary condition can be removed

**Review by shargon — APPROVED** [review]

**Comment by longfeiWan9** [comment]
@erikzhang Erik, can you please help us to merge this PR？

**Pull request merged**